docs/azure_arc_jumpstart/azure_arc_sqlsrv/azure/azure_arm_template_winsrv/_index.md

```diff
@@ -144,84 +152,66 @@ As mentioned, this deployment will use an ARM Template. You will deploy a single
 
-    > **Note: The script run time is ~10-15min long**
-
-    ![Screenshot showing PowerShell script executing in VM](./04.jpg)
-
-    ![Screenshot showing showing PowerShell script executing in VM](./05.jpg)
-
-    ![Screenshot showing showing PowerShell script executing in VM](./06.jpg)
-
-    ![Screenshot showing showing PowerShell script executing in VM](./07.jpg)
-
-    ![Screenshot showing showing PowerShell script executing in VM](./08.jpg)
-
-    ![Screenshot showing showing PowerShell script executing in VM](./09.jpg)
-
-    ![Screenshot showing showing PowerShell script executing in VM](./10.jpg)
-
-    ![Screenshot showing showing PowerShell script executing in VM](./11.jpg)
-
-    ![Screenshot showing showing PowerShell script executing in VM](./12.jpg)
+    > **Note: The script run time is ~30min long**
 
-    ![Screenshot showing showing PowerShell script executing in VM](./13.jpg)
+    ![Screenshot showing PowerShell script executing in VM](./post_deploy_script.png)
 
 * Upon successful run, in the Azure portal, notice you now have a new Azure Arc-enabled server (with the Microsoft Monitoring agent installed via an extension) and Azure Arc-enabled SQL resources as well as Azure Log Analytics added to the resource group.
 
-    ![Screenshot showing Azure Arc-enabled SQL resources](./14.jpg)
+    ![Screenshot showing Azure Arc-enabled SQL resources](./post_deployment_portal_vm.png)
 
-    ![Screenshot showing Azure Arc-enabled SQL resources](./15.jpg)
+    ![Screenshot showing Azure Arc-enabled SQL resources](./post_deployment_portal_arc.png)
 
-    ![Screenshot showing Azure Arc-enabled SQL resources](./16.jpg)
+    ![Screenshot showing Azure Arc-enabled SQL resources](./post_deployment_portal_arc_sql.png)
 
 * Open Microsoft SQL Server Management Studio (a Windows shortcut will be created for you) and validate the *AdventureWorksLT2019* sample database is deployed as well.
 
-    ![Screenshot showing SQL Management Studio](./17.jpg)
+    ![Screenshot showing SQL Management Studio](./sql_server_management_login.png)
 
-    ![Screenshot showing SQL Management Studio](./18.jpg)
+    ![Screenshot showing SQL Management Studio](./sql_server_management_data.png)
 
 ## Azure SQL Assessment
 
 Now that you have both the server and SQL projected as Azure Arc resources, the last step is complete the initiation of the SQL Assessment run.
 
 * On the SQL Azure Arc resource, click on "Environment Health" followed by clicking the "Download configuration script".
 
-    Since the *LogonScript* run in the deployment step took care of deploying and installing the required binaries, you can safely ignore and delete the downloaded *AddSqlAssessment.ps1* file.
+    Since the *installArcAgentSQL* run in the deployment step took care of deploying and installing the required binaries, you can safely ignore and delete the downloaded *AddSqlAssessment.ps1* file.
```

Comment:
```suggestion
    Since the *installArcAgentSQL* run in the deployment step took care of deploying and installing the required binaries you can safely ignore and delete the downloaded *AddSqlAssessment.ps1* file.
```